src/pages/index.mdx

```diff
@@ -12,17 +9,360 @@ export const disableEditGithub = true
 
-<Libraries />
+It's what's missing between applications and infrastructure automation.
 
-<Providers />
+Oh, and it supports basically any language, like JavaScript, TypeScript, Python, Go, you name it.
 
-<Community />
+<div className="">
+<Libraries minimal />
+</div>
+
+---
+
+## Services
+
+Services are the heart of Nitric apps, they're the entrypoints to your code. They can serve as APIs, websockets, schedule handlers, subscribers and a lot more. You create services by telling Nitric where to look for your code and how to run it.
+
+```yaml {{ label:"nitric.yaml" }}
+name: example
+services:
+    - match: services/*.ts
+      start: npm run dev:services $SERVICE_PATH
+```
+
+You might have one service that handles everything, or a service for each route. It's up to you.
+
+---
+
+## Resources
+
+Resources are the building blocks of your apps. They're the databases, queues, buckets, etc. that your services interact with. If you want a resource, you just ask for it.
+
+<CodeGroup>
+
+```javascript {{ label:"services/example.js" }}
+import { bucket } from '@nitric/sdk'
+
+const profiles = bucket('profiles').allow('read', 'write', 'delete')
+```
+
+```python {{ label:"services/example.py" }}
+from nitric.resources import bucket
+from nitric.application import Nitric
+
+profiles = bucket('profiles').allow('read', 'write', 'delete')
+
+Nitric.run()
+```
+
+```go {{ label:"services/example/main.go" }}
+import (
+	"fmt"
+
+	"github.com/nitrictech/go-sdk/nitric"
+)
+
+func main() {
+	profiles, _ := nitric.NewBucket("profiles").Allow(nitric.BucketRead, nitric.BucketWrite, nitric.BucketDelete)
+
+	if err := nitric.Run(); err != nil {
+		fmt.Println(err)
+	}
+}
+```
+
+```dart {{ label:"services/example.dart" }}
+import 'package:nitric_sdk/nitric.dart';
+
+void main() {
+  final profiles = Nitric.bucket("profiles").allow([
+    BucketPermission.read,
+    BucketPermission.write,
+    BucketPermission.delete,
+  ]);
+}
+```
+
+</CodeGroup>
+
+Nitric collects everything your services request. When you deploy, the deployment plugin you choose creates the resources and services, then links it all together.
+
+<Note>
+  That probably sounds like magic, but it's more like a compiler for Infrastructure-as-Code - you can read about it in the [Concepts](/docs/concepts) section.
+</Note>
+
+---
+
+## Providers
+
+Nitric is designed to be independent of any platform, so you can run your apps anywhere. You can deploy to AWS, Azure, GCP, or even your own Kubernetes cluster. You can even deploy to multiple clouds at once.
+
+Seriously, is there really a difference between a bucket on AWS and a bucket on Azure? We don't think so. So why should the code be different?
+
+Nitric abstracts away the differences, so you can focus on your app. The part that makes that possible is a plugin, we call a **Provider**.
```

Comment:
I am not sure how to write it, but since we say there is no difference on the sentence above, perhaps this should read that Nitric abstracts away the cloud specific interface so that you get the same functionality?